morse_code_decoder.rb

```diff
@@ -0,0 +1,53 @@
+@dictionary = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def get_letter(letter)
+  @present_letter = @dictionary[letter]
+  @present_letter
+end
+
+def decode_word(word)
+  @letters = word.split
+  @current_word = ''
+  @letters.each do |letter|
+    get_letter(letter)
+    @current_word += get_letter(letter)
+  end
+  print @current_word
+end
+
+def decode_message(message)
+  @words = message.split
+  @words.each do |word|
+    decode_word(word)
+  end
+end
```

Comment:
- [ ] There seems to be a bug 🐛 with your `decode_message` method as it doesn't work as expected.
For instance, `p decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")` or `decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")` should return "A BOX FULL OF RUBIES" but instead it returns the message without spaces between words and a bunch of characters as well. Please try to fix it 👇 
![image](https://github.com/Fombi-Favour/morse-code-rb/assets/46839548/bb6e1a08-8bc0-4f97-8c13-5fd7df2e057e)